start.py

```diff
@@ -1,21 +1,45 @@
-import subprocess
 import argparse
+import subprocess
+import sys
+
+
+'''
+exit_with_error prints an error to stderr and exits
+
+Parameters
+    error_message(str)
+'''
+
+
+def exit_with_error(error_message):
+    print(error_message, file=sys.stderr)
+    exit(1)
+
 
 commands = {
-    'all': "export FLASK_APP=app.py && flask run",
     'run-windows': "set FLASK_APP=app.py && flask run",
-    'install': "pip3 install -r requirements.txt && cd applications-frontend && npm install && cd ..",
+    'run-other': "export FLASK_APP=app.py && flask run",
+    'install': "pip3 install -r requirements.txt",
     'format': "autopep8 --in-place --aggressive --aggressive *.py",
-    'deploy': "pm2 start start.sh",
-    'publish-deps': 'pip3 freeze > requirements.txt'
-}
+    'deploy': "pm2 start deploy.sh",
+    'save-deps': 'pip3 freeze > requirements.txt',
+    'install-npm': "cd applications-frontend && npm install && cd .."}
```

Comment:
Can we call this `install-client` or something similar?

Reply:
ok